src/status_im/ui/screens/home/sheet/views.cljs

```diff
@@ -73,8 +73,18 @@
      :icon-color                 (colors/theme-colors colors/neutral-50 colors/neutral-40)
      :accessibility-label        :start-a-new-chat
      :icon                       :i/new-message
-     :on-press                   #(hide-sheet-and-dispatch [:bottom-sheet/show-sheet
-                                                            :start-a-new-chat])}]
+     :on-press                   (fn [] ;; Note: currently the bottom button is overlapping with
```

Comment:
can we create an issue to properly address this? This seems like a temporary fix to me.